### PR TITLE
add apiutils.GetTime to get /_meta/time

### PR DESCRIPTION
**Pull request by philipatl**

### CODE REVIEW

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "ff80a51d540706dec259c60ec5306ae9038c558a",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "8afe9d6ec2ba600a78e6f11decac7d445d69f2ce",
    "pipeline": "master"
  }
]
```

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "8afe9d6ec2ba600a78e6f11decac7d445d69f2ce",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "8d979f23b1e81b8c08e2788e720fb2c316e9b674",
    "pipeline": "master"
  }
]
```

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "b8fa152f951236265ff9bf81daba380c3e90d1f8",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "8afe9d6ec2ba600a78e6f11decac7d445d69f2ce",
    "pipeline": "master"
  }
]
```

File: apiutils/utils.go [hunk 1]
```diff
@@ -291,6 +292,44 @@ func GetGoogleOAuthClientID(ctx context.Context, api string, tlsConfig *tls.Conf
 	return ioutil.ReadAll(resp.Body)
 }
 
+// GetTime returns the current time from the api server.
```
Comment (t00f) [inline apiutils/utils.go:295]: We need unittests @philipatl 🙏

**Review by t00f — CHANGES_REQUESTED** [review]

File: apiutils/utils.go (hunk 1 above)
Comment (philipatl) [reply apiutils/utils.go:295]: @t00f Are there unit tests for similar functions here?

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "f3eb3aeaf60b2647dcf18d7aefb7926906893ae0",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "8afe9d6ec2ba600a78e6f11decac7d445d69f2ce",
    "pipeline": "master"
  }
]
```

File: apiutils/utils.go (hunk 1 above)
Comment (primalmotion) [reply apiutils/utils.go:295]: Nope. Since you are the first one to touch that in years, you won the right to write them :)

**Review by primalmotion — DISMISSED** [review]
Minus the tests LGTM. I'll let you add them and let chris merge when he feels like it.

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "8afe9d6ec2ba600a78e6f11decac7d445d69f2ce",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "3ee71fae9a8a0da5344b2261f2867d0d8a59d3a3",
    "pipeline": "master"
  }
]
```

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "0d2484ac2737dcfcd7afc2dbf2ca9d05dcc72c0b",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "3ee71fae9a8a0da5344b2261f2867d0d8a59d3a3",
    "pipeline": "master"
  }
]
```

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "1decac9333e5a7fbbf2c003791a7bea7d2f9f307",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "0d2484ac2737dcfcd7afc2dbf2ca9d05dcc72c0b",
    "pipeline": "master"
  }
]
```

**Comment by philipatl** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "2327",
    "commit-sha": "6187f16f3b7a4a05c5afe473b2d5110378d0c36c",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "addedeffect",
    "pr-id": "79",
    "commit-sha": "0d2484ac2737dcfcd7afc2dbf2ca9d05dcc72c0b",
    "pipeline": "master"
  }
]
```

**Review by t00f — APPROVED** [review]

**Review by primalmotion — APPROVED** [review]
Thank you for the unit tests!

**Pull request merged**